docs/build/wasp/0.7.0/docs/guide/evm/quickstart.mdx

```diff
@@ -29,6 +29,17 @@ Click this button:
 
 <AddToMetaMaskButton />
 
+In case you want to setup manually, here's the details:
+
+| Network Config | Testnet |
+|---|---|
+| JSONRPC | `https://json-rpc.evm.testnet.shimmer.network` |
+| Chain ID | 1072 |
+| Explorer | `https://explorer.evm.testnet.shimmer.network` |
+| Chain Address | `rms1pr75wa5xuepg2hew44vnr28wz5h6n6x99zptk2g68sp2wuu2karywgrztx3` |
+| Token Symbol | RMS |
+| Faucet | https://evm-toolkit.evm.testnet.shimmer.network |
+
```

Comment:
```suggestion
```

Please let's not repeat old mistakes and repeat this info everywhere. This info is already mentioned int the linked detailed guidance